lib/inv_cg_quda.cpp

```diff
@@ -51,7 +51,7 @@ namespace quda {
   }
 
   CGNE::CGNE(DiracMatrix &mat, DiracMatrix &matSloppy, SolverParam &param, TimeProfile &profile) :
-    CG(mmdag, mmdagSloppy, param, profile), mmdag(mat.Expose()), mmdagSloppy(mat.Expose()),
+    CG(mmdag, mmdagSloppy, param, profile), mmdag(mat.Expose()), mmdagSloppy(matSloppy.Expose()),
     xp(nullptr), yp(nullptr), init(false) {
```

Comment:
clang-format suggested the following change:


```
-  CGNE::CGNE(DiracMatrix &mat, DiracMatrix &matSloppy, SolverParam &param, TimeProfile &profile) :
-    CG(mmdag, mmdagSloppy, param, profile), mmdag(mat.Expose()), mmdagSloppy(matSloppy.Expose()),
-    xp(nullptr), yp(nullptr), init(false) {
+  CGNE::CGNE(DiracMatrix& mat, DiracMatrix& matSloppy, SolverParam& param, TimeProfile& profile)
+      : CG(mmdag, mmdagSloppy, param, profile)
+      , mmdag(mat.Expose())
+      , mmdagSloppy(matSloppy.Expose())
+      , xp(nullptr)
+      , yp(nullptr)
+      , init(false)
+  {
```

<!-- identifier: clang-format -->
<!-- File: lib/inv_cg_quda.cpp -->